client/src/components/CreateNewPost.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+
+class CreateNewPost extends React.Component {
+    constructor(props){
+        super(props);
+        this.state = {
+            title: '',
+            contents: '',
+        }
+    }
+
+    handleChange = event => {
+        event.preventDefault();
+        this.setState({
+            [event.target.name]: event.target.value,
+        })
+    }
+
+    handleSubmit = event => {
+        event.preventDefault();
+        this.props.handleAddNewPost({...this.state})
+        //this.setState({[event.target.value]: ''});    not working
```

Comment:
With this you have to do it individually
```suggestion
        this.setState({title: '', contents: ''});
```